### PR TITLE
Add unit tests using sandbox

### PR DESCRIPTION
**Pull request by meraj-work**

### CODE REVIEW

**Review by alexandremcosta — APPROVED** [review]
very well done, thank you!

File: test/ecto_plus_one_test.exs [hunk 1]
```diff
@@ -1,8 +1,52 @@
...
 
-  test "greets the world" do
-    assert EctoPlusOne.hello() == :world
+  import Ecto.Query
+  import ExUnit.CaptureLog
+
+  alias EctoPlusOne.User
+  alias EctoPlusOne.Post
+
+  @repo Application.get_env(:ecto_plus_one, :repo)
```
Comment (alexandremcosta) [inline test/ecto_plus_one_test.exs]: I would just `alias EctoPlusOne.Repo`...but not really important

File: test/ecto_plus_one_test.exs [hunk 2]
```diff
@@ -1,8 +1,52 @@
...
+             # dummy query to trigger N+1
+             from(u in User) |> @repo.all()
+           end) =~ "[warning] N+1 Query Detected!"
+  end
+
+  defp create_user(params) do
+    %User{}
+    |> User.changeset(params)
+    |> @repo.insert()
+  end
```
Comment (alexandremcosta) [inline test/ecto_plus_one_test.exs:45]: You don't have to really create the users. Not even the posts.
A simple ```Enum.each(1..5,  fn -> from(u in User, where: u.id == 1) |> @repo.one() end)```
would do the job...

File: test/test_helper.exs [hunk 3]
```diff
@@ -1 +1,6 @@
 ExUnit.start()
+
+alias EctoPlusOne.Repo
+
+{:ok, _} = Ecto.Adapters.Postgres.ensure_all_started(Repo, :temporary)
```
Comment (alexandremcosta) [inline test/test_helper.exs]: just curious why is this necessary here and not on other test_helpers, for example not in naboo...

File: test/ecto_plus_one_test.exs (hunk 1 above)
Comment (meraj-work) [reply test/ecto_plus_one_test.exs]: changing

File: test/ecto_plus_one_test.exs (hunk 2 above)
Comment (meraj-work) [reply test/ecto_plus_one_test.exs:45]: keeping for now

File: test/test_helper.exs (hunk 3 above)
Comment (meraj-work) [reply test/test_helper.exs]: this is not necessary it turns out. removing
